src/data/specs/Composite.ts

```diff
@@ -160,6 +160,13 @@ export default class Composite extends Specs<CompositeData> {
     })
   }
 
+  ungyrate() {
+    return this.withData({
+      gyrate: (this.data.gyrate - 1) as Count,
+      align: "meta",
+    })
+  }
+
```

Comment:
…I don't actually like this. Since each of these functions is only used once, I'd rather just have it in the actual operations.